sql/core/src/test/scala/org/apache/spark/sql/sources/InsertSuite.scala

```diff
@@ -142,17 +142,6 @@ class InsertSuite extends DataSourceTest with SharedSparkSession {
     )
   }
 
-  test("SELECT clause generating a different number of columns is not allowed.") {
```

Comment:
it should be allowed.

Reply:
should we keep the test, but change its result to assert that it succeeds? or is this behavior exercised elsewhere in this test file?